src/store/flow/flow.ts

```diff
@@ -332,7 +333,11 @@ export const actions: ActionTree<IFlowsState, IRootState> = {
     }
 
     commit('flow_addBlock', { block: duplicatedBlock })
-    commit('builder/activateBlock', { blockId: duplicatedBlock.uuid }, { root: true })
```

Comment:
Wondering why we deleted this line ? This is telling the store the new duplicated block is the activeBlock, meaning this block will be highlighted in the canvas

Reply:
Deleted this line as we want the block to be select due to the route change. When the route changes with a block id that block will be automatically selected (these changes can be seen below in InteractionDesigner.vue file).

Reply:
Ok